.husky/pre-commit

```diff
@@ -1,14 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-echo '🏗️👷 Testing the build before committing'
-
-yarn test-all ||
-(
-    echo '\n\n❌👷🔨❌ Build Failed ❌👷🔨❌
-            Next build failed: View the errors above to see why.
-    '
-    false;
-)
-
-echo '✅✅✅✅ Production build successful ✅✅✅✅'
+npm test
```

Comment:
Restore the previous file. Change `yarn` -> `npm run`

Reply:
If I changed it manually, then it will automatically change it back to npm test after someone install the packages, that is was the prepare command is doing.

Reply:
can we remove the prepare command, then it will not run the script, and the user will also get the pre-commit that is already available in the repo.